masu/external/date_accessor.py

```diff
@@ -39,20 +39,23 @@ class DateAccessor():
     """Accessor to get date time."""
 
     mock_date_time = None
-    date_time_last_accessed = datetime.now()
+    date_time_last_accessed = datetime.now(tz=pytz.UTC)
 
     def __init__(self):
         """Initializer."""
         if Config.MASU_DATE_OVERRIDE and Config.DEBUG and DateAccessor.mock_date_time is None:
-            DateAccessor.mock_date_time = datetime.strptime(Config.MASU_DATE_OVERRIDE,
-                                                            '%Y-%m-%d %H:%M:%S')
```

Comment:
Parser provides flexibility in the face of uncertainty. We know the exact format of our override timestamp. Going by Zen of Python "explicit is better than implicit", I think we should continue to use the explicit `strptime`.

Reply:
I agree with you on the point of implicit versus explicit. 

However, there's also the principle of "standards-based is better than making something up on your own."

My main motivation for using dateutil is that I know it supports ISO8601 formatting. I know that `datetime.fromisoformat()` isn't available unless we move to Python 3.7. I can look around for a different, explicit option. But, part of my goal is dropping our homegrown formatting in favor of shifting over to the ISO8601 format as what we are going to support.